internal/indexmigration/migration.go

```diff
@@ -49,11 +49,11 @@ func Migrate(paths environment.Paths) error {
 		return nil
 	}
 
-	err = os.RemoveAll(paths.IndexPath(""))
+	err = os.RemoveAll(paths.IndexPath("default"))
 	if err != nil {
-		return errors.Wrapf(err, "could not remove index directory %q", paths.IndexPath(""))
+		return errors.Wrapf(err, "could not remove index directory %q", paths.IndexPath("default"))
 	}
-	err = gitutil.EnsureCloned(constants.IndexURI, filepath.Join(paths.IndexPath(""), "default"))
+	err = gitutil.EnsureCloned(constants.IndexURI, filepath.Join(paths.IndexBase(), "default"))
```

Comment:
Can you add a todo comment here that `filepath.Join(paths.IndexBase(), "default")` be replaced with `paths.IndexPath("default")` at some point?

Or even better, can we use that right now? After  all, migration should only be called when the env var is set and then `IndexPath("default")` will give the correct result.